src/resources/ui/indentation1.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0" y="0" width="40" height="40" viewBox="0 0 40 40" enable-background="new 0 0 40 40" xml:space="preserve">
+  <g stroke="#aaaaaa"  stroke-width="1" width="40" height="40">
+    <line x1="20" y1="0" x2="20" y2="40"/>
+  </g>
+</svg>
```

Comment:
Can we rename these three files to `indent-line-i.svg`, `t.svg` and `l.svg`, respectively?